site/source/tutorials/1030_routing/index.md

```diff
@@ -76,51 +78,62 @@ Next, we will create `outlets` to control when our widgets are displayed.
 
 {% section %}
 
-## Routing outlets
+## Routing Outlets
 
-{% task 'Create outlets for the routed widgets.' %}
+{% task 'Wrap the application widgets with the `Outlet` widget.' %}
 
 {% aside 'Reminder' %}
 The path that is associated to an outlet name is defined by the routing configuration from the first section of this tutorial.
 {% endaside %}
 
-An `Outlet` is a higher order component that wraps a widget, and controls whether the widget is rendered based on the navigated route associated to the outlets name. To keep our application organized, outlets are usually stored in a separate directory named `outlets`.
+The `Outlet` widget accepts two properties `id` and `renderer`, the `id` is the `outlet` from the routing configuration and the `renderer` is a function that returns widgets and nodes to display using `v()` and `w()`.
```

Comment:
two properties `id` -> two properties, `id` 

`renderer`, the `id` is the -> `renderer`. The `id` is the